README.md

```diff
@@ -173,11 +173,13 @@ In order to speed up testing, Barista keeps in mind some considerations.
 # Download
 
 ```gradle
-androidTestCompile('com.schibsted.spain:barista:1.1.0') {
+androidTestCompile('com.schibsted.spain:barista:x.y.z') {
```

Comment:
Hahahaha that's true. Try to figure how to change the README.md too. We can do some sort of commits, replacing the old version with the new one. What do you think, @alorma ?